Includes/xbe.h

```diff
@@ -0,0 +1,25 @@
+#ifndef XBE_H
+#define XBE_H
+
+#define XBETYPESTRING 0x48454258
+#define XBENAMESIZE 0x028
```

Comment:
nit: Why hex? Why 3 digits? This would imply a 12 bit value that's used for addressing or other non-human use. 40 is much easier to interpret here.